syntax = "proto3";

package request;

enum RequestType {
	UNKNOWN = 0;
    QUERY_AUTH = 1;
    QUERY_USERINFO = 2;
    UPDATE_AUTH = 3;
    UPDAT_PRODUCT = 4;
}

message RequestData {
    RequestType type = 1;
    string cmd = 2;
    map<string, string> extra = 3;
    map<string, string> ret = 4;
}
    